components/wpt-results.html

```diff
@@ -46,59 +120,170 @@
 
       static get properties() {
         return {
+          query: {
+            type: String,
+            value: '',
+            observer: '_queryChanged'
+          },
           specs: {
             type: Array,
-            observer: '_specsChanged',
             value: []
-          }
+          },
+          testRuns: {
+            type: Array,
+          },
         };
       }
 
-      connectedCallback() {
+      async connectedCallback() {
         super.connectedCallback();
 
-        this._fetchResults()
+        const testFileResults = await Promise.all(this.testRuns.map(testRun => {
+          return this._fetchResults(testRun.results_url)
+        }))
+
+        let specs = []
+
+        testFileResults.forEach(result => {
+          const testFiles = result.testFiles
+          const resultsURL = result.resultsURL
```

Comment:
Semicolon use should be a project-wide decision - I used to be a fan of semicolons but was converted after using https://standardjs.com/ - specifically the "No semicolons" bullet point on that page gives a lot of good rationale for leaving them out. I'm open to either style and I know that Google leans more toward semicolon usage.